README.md

```diff
@@ -1 +1,303 @@
-# edgemesh
\ No newline at end of file
+# EdgeMesh
+
+English | [简体中文](./README_ZH.md)
+
+
+
+## Introduction
+
+EdgeMesh is closely related to KubeEdge, and provides a simple network solution for the inter-communications between services at edge scenarios.
+
+At present, EdgeMesh does not have a Pod network, so it relies on the connectivity of the host network. In the future, EdgeMesh will realize the capabilities of CNI plug-ins, and realize the Pod network connectivity between edge nodes and nodes on the cloud, or edge nodes across LANs in a  compatible manner with mainstream CNI plug-ins (e.g., flannel / calico, etc). Finally, EdgeMesh can even replace part of its own components with cloud-native components (e.g., replacing [kube-proxy](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/) to achieve the capabilities of the Cluster IP, replacing [node local dns cache ](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to achieve node-level dns capabilities, and replace [envoy](https://www.envoyproxy.io/) to achieve mesh-layer capabilities).
```

Comment:
Please describe the project background, motivation and target position (in the tech stack) here.

Reply:
I have modified this content as required.